src/createServerClient.ts

```diff
@@ -170,11 +163,11 @@ export function createServerClient<
         ...(options?.cookieOptions?.name
           ? { storageKey: options.cookieOptions.name }
           : null),
-        ...options?.auth,
         flowType: "pkce",
```

Comment:
It looks like it'd no longer be possible to override `persistSession` , `autoRefreshToken` etc.. with user supplied options after this change. Not sure if we'd want that

Let me know if I've misunderstood the change though.